spec/filesystem.rb

```diff
@@ -75,6 +75,23 @@ def create_initial_directory_hierarchy
     FileUtils.touch Metalware::Constants::DEFAULT_CONFIG_PATH
   end
 
+  # Print every directory and file loaded in the FakeFS.
+  def debug!
+    begin
+      # This can fail oddly if nothing matches (see
+      # https://github.com/fakefs/fakefs/issues/371), hence the `rescue` with a
+      # simpler glob.
+      matches = Dir['**/*']
```

Comment:
Thanks for the suggestion - this doesn't appear to fix it however; if you add `p Dir['/**/*']` to the first `it` in the issue I opened (https://github.com/fakefs/fakefs/issues/371) it will still raise an error.

I think this is related to the way shell-globbing works, which `Dir.[]` attempts to match, e.g. if you run the following from the Metalware repo root:
```
$ echo src/commands/repo/**/*
src/commands/repo/update.rb src/commands/repo/use.rb
```
- you can see that the directory `src/commands/repo/` itself is not matched by the glob.